README.md

```diff
@@ -35,9 +35,9 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 
 - Make sure to update your operating system, drivers, and DirectX (if applicable) to ensure you have the best experience possible. Having a newer GPU is also recommended so you have the latest supported drivers.
 
-- Because of copyright issues, and the complexity of trying to work around it, you need a BIOS dump extracted from a legitimately-owned PS2 console to use the emulator. For more information about the BIOS and how to get it from your console, visit [this page](https://pcsx2.net/config-guide/official-english-pcsx2-configuration-guide.html#Bios).
+- Because of copyright issues—and the complexity of trying to work around it—you need a BIOS dump extracted from a legitimately-owned PS2 console to use the emulator. For more information about the BIOS and how to get it from your console, visit [this page](https://pcsx2.net/config-guide/official-english-pcsx2-configuration-guide.html#Bios).
```

Comment:
This line didn't need changing.

Reply:
... at least, it should be " - ", not "-" (spaces between the dash sign).